Organiser.Core/Organiser.Core/Controllers/RolesController.cs

```diff
@@ -27,20 +28,20 @@ public RolesViewModel GetUserRoles()
 
             var model = new RolesViewModel()
             {
-                IsAdmin = userRole == 3,
-                IsSupport = userRole == 3 || userRole == 2,
-                IsUser = userRole == 3 || userRole == 2 || userRole == 1,
+                IsAdmin = userRole == (int) RoleEnum.Admin,
+                IsSupport = userRole == (int) RoleEnum.Admin || userRole == (int) RoleEnum.Support,
+                IsUser = userRole == (int) RoleEnum.Admin || userRole == (int) RoleEnum.Support || userRole == (int)RoleEnum.User,
             };
 
             return model;
         }
 
         [HttpGet]
         [Route("GetIsUserAdmin")]
-        public bool GetIsUserAdmin() => context.User.FirstOrDefault(x => x.UID == user.UID)?.URID == 3;
+        public bool GetIsUserAdmin() => context.User.FirstOrDefault(x => x.UID == user.UID)?.URID == (int) RoleEnum.Admin;
 
         [HttpGet]
         [Route("GetIsUserSupport")]
-        public bool GetIsUserSupport() => context.User.FirstOrDefault(x => x.UID == user.UID)?.URID == 2;
+        public bool GetIsUserSupport() => context.User.FirstOrDefault(x => x.UID == user.UID)?.URID == (int) RoleEnum.Support;
```

Comment:
you already have an endpoint for getting user roles, why you need additional ones? I'd suggest to get user roles on app initialization and keep it in a frontend state where you already have state management system in place